nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/PutKinesisStream.java

```diff
@@ -95,8 +94,7 @@ public class PutKinesisStream extends AbstractAwsSyncProcessor<KinesisClient, Ki
             .build();
 
     public static final PropertyDescriptor MAX_MESSAGE_BUFFER_SIZE_MB = new PropertyDescriptor.Builder()
-            .name("max-message-buffer-size")
-            .displayName("Max message buffer size (MB)")
+            .name("Max message buffer size (MB)")
```

Comment:
The name should be changed to standard conventions:
```suggestion
            .name("Max Message Buffer Size")
```